src/sql.rs

```diff
@@ -1250,6 +1250,51 @@ async fn open(
             .await?;
             sql.set_raw_config_int(context, "dbversion", 64).await?;
         }
+        if dbversion < 65 {
+            info!(context, "[migration] v65");
+            sql.execute(
+                "ALTER TABLE chats ADD COLUMN autodelete_timer INTEGER;",
+                paramsv![],
+            )
+            .await?;
+            // Timer value in seconds. For incoming messages this
+            // timer starts when message is read, so we want to have
+            // the value stored here until the timer starts.
+            sql.execute(
+                "ALTER TABLE msgs ADD COLUMN autodelete_timer INTEGER DEFAULT 0;",
+                paramsv![],
+            )
+            .await?;
+            // Timestamp indicating when the message should be
+            // deleted. It is convenient to store it here because UI
+            // needs this value to display how much time is left until
+            // the message is deleted.
+            sql.execute(
+                "ALTER TABLE msgs ADD COLUMN autodelete_timestamp INTEGER DEFAULT 0;",
+                paramsv![],
+            )
+            .await?;
+            sql.set_raw_config_int(context, "dbversion", 65).await?;
+        }
+        if dbversion < 66 {
+            info!(context, "[migration] v66");
```

Comment:
:(

Reply:
We can do it in one step if `dbversion < 65` and only do rename if `dbversion == 65`. I am trying not to break database again for those who test this PR with their primary database.

Reply:
Squashed it back into one migration

Reply:
sorry, i didn't mean you had to change it.  These are super cheap anyway and I understand why it's easier and safer to do.